src/store/actions/session.js

```diff
@@ -22,9 +24,22 @@ import { getUserProfile } from './user'
  * @returns {Function} Redux action thunk
  */
 export const logout = (ctx) => {
-  return (dispatch, getState) => {
+  return async (dispatch, getState) => {
     deleteCookie('access_token', ctx)
 
+    const token = selectSessionToken(getState())
```

Comment:
There's a second call to `getState()` in this same function now, Could you extract that to a separate variable?